services/ui-src/src/containers/PackageList.js

```diff
@@ -233,7 +233,14 @@ const PackageList = () => {
               filter: CustomFilterTypes.DateRange,
               Filter: CustomFilterUi.DateRangeInPast,
             }
-          : null,
+          : {
```

Comment:
I probably would have moved the isCMSUser check to the Header, since all the rest is the same..... but this is less of a code change and wouldn't interfere with what was done for CMS users, so this way is not a bad choice, either.